packages/data-provider/test/utils/dictionarySchemasTestData.ts

```diff
@@ -0,0 +1,264 @@
+import { Dictionary } from '@overture-stack/lyric-data-model';
+import { SchemaDefinition, ValueType } from '@overturebio-stack/lectern-client/lib/schema-entities.js';
+
+const sampleDictionaryData: SchemaDefinition[] = [
+	{
+		name: 'study',
```

Comment:
is it reasonable to suggest we use a more generic test model? study and participant seem PCGL-specific, when we could be using fruits or animals to clearly exemplify the reusability of the software.